keyboards/ergodone/keymaps/via/readme.md

```diff
@@ -0,0 +1,5 @@
+
+## Changelog
+HarshitGoel96
+
+I love my ergodone, but it needed via, so here it is. Flash using HIDBOOTLOADER as described in keyboard root read.md and enjoy via on ergodone.
```

Comment:
```suggestion
I love my ergodone, but it needed via, so here it is. Flash using HIDBOOTLOADER as described in keyboard root readme.md and enjoy via on ergodone.
```
Although I think this file is mostly useless.
Changelog is kept by `git` and flashing is the same as any other keymap on this board.